traits/transaction_summary_id.raml

```diff
@@ -0,0 +1,6 @@
+queryParameters:
+  transaction_summary_id:
+    description: |
+      Required transaction summary id
+    type: UUID
+    required: true
```

Comment:
If it is a required parameter it could also go into the URL:
```
POST /finance-storage/transactions/{transactionSummaryId}
PUT /finance-storage/transactions/{transactionSummaryId}/{id}
```

Reply:
If I saw a request like `POST /finance-storage/transactions/123`, I would be confused about what 123 is for. I guess a more explicit and REST-like alternative would be to use something like:
`POST /finance-storage/transaction-summaries/{transactionSummaryId}/transactions`
`PUT /finance-storage/transaction-summaries/{transactionSummaryId}/transactions/{id}`

Reply:
I agree.